### ISSUE TITLE
не работает tag.cyr_repr

### ISSUE THREAD

**Issue by tytarenko** [kmike/pymorphy2#81]
При использовании укр словоря
p.tag.cyr_repr
возвращает 
`noun,m,inanim v_mis`
вместо 
`ГЛ,сов,неперех мн,прош,изъяв`
или подобному на укр языке

**Comment by kmike** [comment]
Да, там сейчас возвращаются пометки в формате LanguageTool, если не ошибаюсь.
